### PR TITLE
Pr/save values to database

### PR DESCRIPTION
**Pull request by eliatib**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "symfony/symfony": "4.1.8",
+        "symfony/browser-kit": "4.1.8",
+        "symfony/dependency-injection": "4.1.8",
+        "symfony/dom-crawler": "4.1.8",
+        "symfony/routing": "4.1.8"
+    },
+    "prefer-stable": true,
+    "autoload": {
+        "psr-4": {
+            "MangoSylius\\ContactFormPlugin\\": "src/",
```
Comment (ondrej-kuhnel) [inline composer.json]: Better use `SyliusContactFormPlugin`

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+
+* Extend contact form
+* Add Message administrative panel
+    * conversation history
+    * Possibility to respond instantly
+
+## Installation
+
+1. Run `$ composer require mangoweb-sylius/sylius-contact-form-plugin`.
+2. Register `\MangoSylius\ContactFormPlugin\MangoSyliusOrderCommentsPlugin` in your Kernel.
```
Comment (ondrej-kuhnel) [inline README.md]: Update whole readme

File: src/Resources/config/services.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      $entityManager: '@doctrine.orm.entity_manager'
+      $templatingEngine: '@templating'
+      $builder: '@form.factory'
+      $mailer: '@sylius.email_sender'
+      $adminUserRepository: '@sylius.repository.admin_user'
+
+
+
+
+
```
Comment (ondrej-kuhnel) [inline src/Resources/config/services.yml]: Remove blank lines

File: src/Resources/config/services.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+services:
+  _defaults:
```
Comment (ondrej-kuhnel) [inline src/Resources/config/services.yml]: always use 4 spaces

File: src/Form/Type/ContactFormType.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            ])
+            ->add('phone', TelType::class, [
+                'label' => 'mango_sylius.contactForm.phone',
+                'required' => true,
+            ])
+            ->add('message', TextareaType::class, [
+                'label' => '',
+                'attr' => [
+                    'placeholder' => 'mango_sylius.contactForm.message',
+                    'required' => true,
```
Comment (ondrej-kuhnel) [inline src/Form/Type/ContactFormType.php]: wrong location :)
moving it up a level

File: src/Entity/ContactFormMessage.php [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+namespace MangoSylius\ContactFormPlugin\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="app_contact_form")
```
Comment (ondrej-kuhnel) [inline src/Entity/ContactFormMessage.php]: Start with bundle name

File: src/Entity/ContactFormMessage.php [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+     * @Assert\NotBlank
+     */
+    protected $message;
+
+    /**
+     * @ORM\Column(type="datetime")
+     *
+     * @var \DateTime|null
+     */
+    protected $sendTime;
```
Comment (ondrej-kuhnel) [inline src/Entity/ContactFormMessage.php]: better name `createdAt`

File: src/Entity/ContactFormMessage.php [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="app_contact_form")
+ */
+class ContactFormMessage implements ResourceInterface
```
Comment (ondrej-kuhnel) [inline src/Entity/ContactFormMessage.php:15]: Add column IP

File: src/Controller/ContactFormController.php [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+        ]);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            if ($form->isValid()) {
+                $contact->setSendTime(new \DateTime());
+                $this->entityManager->persist($contact);
+                $this->entityManager->flush();
+                $SM = $this->adminUserRepository->find(1);
```
Comment (ondrej-kuhnel) [inline src/Controller/ContactFormController.php]: Why this?
Remove it :)

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (eliatib) [reply composer.json]: DONE

File: README.md (hunk 2 above)
Comment (eliatib) [reply README.md]: DONE

File: src/Resources/config/services.yml (hunk 3 above)
Comment (eliatib) [reply src/Resources/config/services.yml]: DONE

File: src/Resources/config/services.yml (hunk 4 above)
Comment (eliatib) [reply src/Resources/config/services.yml]: DONE

File: src/Form/Type/ContactFormType.php (hunk 5 above)
Comment (eliatib) [reply src/Form/Type/ContactFormType.php]: DONE

File: src/Entity/ContactFormMessage.php (hunk 6 above)
Comment (eliatib) [reply src/Entity/ContactFormMessage.php]: DONE

File: src/Entity/ContactFormMessage.php (hunk 7 above)
Comment (eliatib) [reply src/Entity/ContactFormMessage.php]: DONE

File: src/Entity/ContactFormMessage.php (hunk 8 above)
Comment (eliatib) [reply src/Entity/ContactFormMessage.php:15]: DONE

File: src/Controller/ContactFormController.php (hunk 9 above)
Comment (eliatib) [reply src/Controller/ContactFormController.php]: DONE

File: src/Resources/config/services.yml (hunk 4 above)
Comment (ondrej-kuhnel) [reply src/Resources/config/services.yml]: in every *.yml and *.yaml file

File: src/Resources/config/services.yml (hunk 3 above)
Comment (ondrej-kuhnel) [reply src/Resources/config/services.yml]: in every *.yml and *.yaml file

File: src/Entity/ContactFormMessage.php (hunk 6 above)
Comment (ondrej-kuhnel) [reply src/Entity/ContactFormMessage.php]: change `app` to `mango`

File: src/Controller/ContactFormController.php [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+    /** @var FormFactoryInterface */
+    private $builder;
+    /** @var UserRepositoryInterface */
+    private $adminUserRepository;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        EngineInterface $templatingEngine,
+        EntityManagerInterface $entityManager,
+        SenderInterface $mailer,
```
Comment (ondrej-kuhnel) [inline src/Controller/ContactFormController.php]: Remove all unused variables

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: src/Resources/config/services.yml (hunk 4 above)
Comment (eliatib) [reply src/Resources/config/services.yml]: DONE

File: src/Entity/ContactFormMessage.php (hunk 6 above)
Comment (eliatib) [reply src/Entity/ContactFormMessage.php]: DONE

File: src/Controller/ContactFormController.php (hunk 10 above)
Comment (eliatib) [reply src/Controller/ContactFormController.php]: DONE

File: src/Resources/config/services.yml (hunk 3 above)
Comment (eliatib) [reply src/Resources/config/services.yml]: DONE

**Review by ondrej-kuhnel — APPROVED** [review]
squash it and merge

**Pull request merged**